provider/iam/token_exchange.go

```diff
@@ -117,13 +125,15 @@ func (tes *tokenExchangeService) ExchangeIAMAPIKeyForIMSToken(iamAPIKey string,
 }
 
 // ExchangeIAMAPIKeyForAccessToken ...
-func (tes *tokenExchangeService) ExchangeIAMAPIKeyForAccessToken(iamAPIKey string, logger *zap.Logger) (*AccessToken, error) {
-	r := tes.newTokenExchangeRequest(logger)
-
-	r.request.Field("grant_type", "urn:ibm:params:oauth:grant-type:apikey")
-	r.request.Field("apikey", iamAPIKey)
-
-	return r.exchangeForAccessToken()
+func (tes *tokenExchangeService) ExchangeIAMAPIKeyForAccessToken(logger *zap.Logger) (*AccessToken, error) {
```

Comment:
Any plan to change function name ? or due to many providers using this are we keeping same name ?

Reply:
Since, this is used by file vpc also, won't be changing the name currently.